assets/src/edit-story/components/button/index.js

```diff
@@ -178,3 +179,9 @@ export const More = forwardRef((props, ref) => (
     <MoreIcon />
   </StyledButton>
 ));
+
+export const SafeMode = (props) => (
```

Comment:
While it's just a passthrough to StyledButton with an icon let's still add some prop types here that StyledButton would require. Likely can just assign to it's propTypes.